test/unit/DCAStrategies/dca-strategies-management-handler.spec.ts

```diff
@@ -27,6 +28,18 @@ contract('DCAStrategiesManagementHandler', () => {
     await snapshot.revert(snapshotId);
   });
 
+  function generateRandomAddress() {
```

Comment:
I think we already have a utils in utils/wallet that does this

Reply:
Yes, but I wanted different addresses on each call. Is it mandatory that the tests be deterministic?

Reply:
I think it wasn't because of determinism, it was so that the gas cost didn't change each time the test was run. But this is good enough for me